concent_api/middleman/middleman_server.py

```diff
@@ -127,6 +127,8 @@ def _close_middleman(self) -> None:
             await self._respond_to_user(frame, writer)
         except (BrokenEscapingInFrameMiddlemanProtocolError, asyncio.LimitOverrunError, MiddlemanProtocolError) as exception:
             await self._send_immediate_error(writer, exception)
+        except asyncio.streams.IncompleteReadError:
+            pass
```

Comment:
This should be handled in the decorator, not here. It's not an error like the other exceptions being caught here.

Have you read what I wrote in the issue?